### DISCUSSION TITLE
Issue Triaging

### DISCUSSION THREAD

**Discussion by mefellows** [pact-foundation/roadmap#15] [Ideas]
As part of the [SmartBear Supported](https://github.com/pact-foundation/devrel/discussions/12) initiative and as an outcome of our [2023 Strategy Workshops](https://github.com/pact-foundation/devrel/discussions/13), we identified a need to improve issue triaging across the Pact estate.

**Objectives**

* Ensure contributions are regularly reviewed and responded to in a timely manner
* Reduce the cost of managing projects across the ecosystem by improving process and automation 
* Identify work that SmartBear can take ownership of to accelerate roadmap and bug fixes

This discussion has been created to break down elements of this initiative.

Key components:

| Item | Description | Link | Owner | Status | 
|-----|-------------|-----|--------|--------|
| Issue Triage Board | Create a project to track incoming issues and pull requests, to be able to meet SLAs for defined projects, and encourage movement for contributions. Project will be public for transparency. | [link](https://github.com/orgs/pact-foundation/projects/16/views/1) | @mefellows @YOU54F | 🟠 In progress  |
| Issue Triage Process | Define process to triage new issues, including frequency of review, workflow etc. | [link](https://github.com/pact-foundation/devrel/discussions/16) | @mefellows @YOU54F | 🟠 In progress  |
| Standardise Labels | Create a standard set of issue labels (and descriptions) all projects can use, and automate responses via GitHub bots/actions | [link](https://github.com/pact-foundation/devrel/discussions/17) | @mefellows @YOU54F  | 🟠 In progress 
| Common Issue Templates | Create a standard set of issue and feature request templates all projects can use. | [link](https://github.com/pact-foundation/devrel/discussions/18) | @mefellows @YOU54F | 🟠 In progress  |
| Define SLAs | Agree on what the initial SLAs look like (probably just ticket response time to start simple!) | link | @mefellows @YOU54F | 🟠 In progress  |

**Comment by mefellows** [comment]
via @JP-Ellis :

> With the automation of labels, this is something I’ve used quite successfully:
> GitHub Action: https://github.com/JP-Ellis/tikz-feynman/blob/develop/.github/workflows/labels.yml
> Label Definition: https://github.com/JP-Ellis/tikz-feynman/blob/develop/.github/labels.yml

**Reply by JP-Ellis** [reply]
To further expand on the above (and I understand this is an implementation detail) but [EndBug's `label-sync` action](https://github.com/EndBug/label-sync) can make reference to multiple label definitions. This would easily allow a set of common labels to be applied across all repos, and then each individual repo can create additional labels if need be.

**Reply by mefellows** [reply]
That looks really handy. I could imagine wanting to use labels for project tracking etc., and that would be independent of the "global" standardd.
